README.md

```diff
@@ -86,13 +88,17 @@ exfiltrate that data, and Client Hints only provides them with a more
 convenient and performant way to do that, when that data is needed for content
 negotiation purposes.
 
-But, that also means that we don't want passive subresources (e.g. images) to
-be able to exfiltrate the same data, and we certainly don't want them to be
+[TODO: mention that CSS can also probe.. not just JavaScript]
+
+But, that also means that we don't want passive 3P subresources (e.g. images) to
+be able to exfiltrate the same data without explicit permission, and we certainly don't want them to be
 able to exfiltrate it for the entire origin for a longer amount of time.
 
-Therefore, by default, client hints are only applicable to the same origin as
-the page navigation, and the opt-in is only available for navigation requests,
-which are potentially active resources.
+[TODO ^ what does longer amount of time mean?]
+
+Therefore, by default, Client Hints opt-in is only applicable to 1P resources, and access by 3P subresources must be explicitly permitted by the 1P origin.
+
+[TODO ^ unless I'm misintrepreting what you were trying to explain here?]
```

Comment:
Yeah, my phrasing can be better here. I was trying to explain why the opt-in itself is something that only works when delivered over top-level navigation responses.